db/schema.rb

```diff
@@ -612,6 +612,7 @@
     t.string "encrypted_password_digest_multi_region"
     t.string "encrypted_recovery_code_digest_multi_region"
     t.datetime "second_mfa_reminder_dismissed_at"
+    t.datetime "sign_in_new_device"
```

Comment:
This doesn't follow the typical naming convention for datetime columns suffixed with `_at` to indicate when something occurred.

Reply:
That is now corrected to use _at